recipes/bob.ip.flandmark/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [[ `uname` == 'Darwin' ]]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.9
+    export CFLAGS="${CFLAGS} -pthread"
+    export LDFLAGS="${LDFLAGS} -lpthread"
+fi
```

Comment:
This should be replaced. Please see this PR ( https://github.com/conda-forge/thrift-cpp-feedstock/pull/4 ) for how to change this.